homeassistant/components/sensor/xbox_live.py

```diff
@@ -97,6 +99,8 @@ def device_state_attributes(self):
                 attributes[
                     '{} {}'.format(device.get('type'), title.get('placement'))
                 ] = title.get('name')
+                attributes['gamerscore'] = self._gamerscore
```

Comment:
Please unindent the new attributes out of the nested for loop. These attributes are the same per entity.

Reply:
@MartinHjelmare Sorry about that, it's fixed now.